server/http_integration_test.go

```diff
@@ -282,8 +282,7 @@ func TestHTTPServer_ServeHTTP(t *testing.T) {
 				entry := logHook.LastEntry()
 
 				if entry == nil || entry.Data["type"] != "couper_access" {
-					t.Error("Expected a log entry, got nothing")
-					return
+					t.Fatal("Expected a log entry, got nothing")
```

Comment:
Now all running subtests ends because of one missing entry. Cleanup call is missing too. 
This should be reverted.